### PR TITLE
Add necessary files

### PR DESCRIPTION
**Pull request by NalaGinrut**
Add these files:

- .gitignore
- CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+* Create a topic branch from where you want to base your work.
+* This is usually the master branch.
+* To quickly create a topic branch based on master, run `git checkout -b
+fix/master/my_contribution master`. Please avoid working directly on the
+`master` branch.
+* Make commits of logical units.
+* Check for unnecessary whitespace with `git diff --check` before committing.
+* Make sure your commit messages are in the proper format. If the commit
+addresses an issue filed in the
+[ch554_sdcc Issues](https://github.com/Blinkinlabs/ch554_sdcc/issues), start
```
Comment (cibomahto) [inline CONTRIBUTING.md]: Should reference the current project

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+the new documentation or comments added.
+```
+
+For commits that address trivial repository maintenance tasks, start the
+first line of the commit with `(maint)`.
+
+## Submitting Changes
+
+* Push your changes to a topic branch in your fork of the repository.
+* Submit a pull request to the repository in the Blinkinlabs organization.
```
Comment (cibomahto) [inline CONTRIBUTING.md]: Change 'Blinkinlabs' to a generic 'this organization'

**Review by cibomahto — CHANGES_REQUESTED** [review]
Request a few name changes in the contributing document.

File: CONTRIBUTING.md (hunk 1 above)
Comment (NalaGinrut) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 2 above)
Comment (NalaGinrut) [reply CONTRIBUTING.md]: done

**Review by cibomahto — APPROVED** [review]

**Pull request merged**